packages/pluggable-widgets-tools/test-config/jest.config.js

```diff
@@ -4,28 +4,25 @@ const projectDir = process.cwd();
 
 module.exports = {
     clearMocks: true,
+    testRunner: "jest-jasmine2",
     rootDir: join(projectDir, "src"),
     setupFilesAfterEnv: [join(__dirname, "test-index.js")],
     snapshotSerializers: ["enzyme-to-json/serializer"],
+    testEnvironment: "jsdom",
```

Comment:
testEnvironment is already configured at the bottom of this file

Reply:
Something went wrong with my rebase